src/main/java/org/testng/internal/reflect/ReflectionHelper.java

```diff
@@ -1,14 +1,12 @@
 package org.testng.internal.reflect;
 
-import java.util.Arrays;
 import java.util.stream.Collectors;
 import org.testng.collections.Lists;
 
 import java.lang.annotation.Annotation;
 import java.lang.reflect.Method;
 import java.lang.reflect.Modifier;
-import java.util.LinkedList;
-import java.util.List;
+import java.util.*;
```

Comment:
TestNG codebase follows a convention of not using wildcard imports. Can you please help revert this ?

Reply:
Did not know that, fixed!